spec/function.dd

```diff
@@ -190,6 +205,59 @@ $(H2 $(LNAME2 contracts, Contracts))
         {
             ...contract preconditions...
         }
+        do
+        {
+            ...function body...
+        }
+        ---
+
+
+    $(H3 Postconditions)
+
+        $(P An $(GLINK OutContractExpression) is a postcondition.)
+
+        $(P The first $(GLINK2 expression, AssignExpression) of the $(GLINK2 expression, AssertArguments)
+        must evaluate to true. If it does not, the postcondition has failed.)
+
+        $(P The second $(I AssignExpression), if present, must be implicitly convertible to type `const(char)[]`.
+        )
+
+        $(P An $(GLINK OutStatement) is also a postcondition. Any $(GLINK2 expression, AssertExpression) appearing
+        in an $(OutStatement) will be an $(I OutContractExpression).
```

Comment:
boom `[UNDEFINED MACRO]         in an <!--UNDEFINED MACRO: "OutStatement"--> will be an <i>OutContractExpression</i>`